containers/lean/Dockerfile

```diff
@@ -4,13 +4,14 @@ RUN \
   apk update && apk upgrade && \
 
   # install ruby
-  apk --no-cache add ruby ruby-dev ruby-bundler ruby-json ruby-irb ruby-rake ruby-bigdecimal && \
+  apk --no-cache add ruby ruby-dev && \
```

Comment:
is there a notable difference if the `ruby` and `ruby-dev` packages get removed/purged from the container at the end?